addons/website/static/src/js/tours/tour_utils.js

```diff
@@ -195,6 +195,29 @@ function clickOnEditAndWaitEditMode(position = "bottom") {
     }];
 }
 
+/**
+ * Click on the top right edit dropdown, then click on the edit dropdown item
+ * and wait for the edit mode
+ *
+ * @param {string} position Where the purple arrow will show up
+ */
+function clickOnEditAndWaitEditModeInTranslatedPage(position = "bottom") {
+    return [{
+        content: markup(_t("<b>Click Edit</b> dropdown")),
+        trigger: ".o_edit_website_container button",
+        position: position,
+    }, {
+        content: markup(_t("<b>Click Edit</b> to start designing your homepage.")),
```

Comment:
@odoo/rd-security While this PR is getting reviewed, would you mind already overriding the `ci/security` triggered by those 2 `markup()`?
Those are safe, we just inject our own hardcoded `<b/>`.